NAMESPACE

```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export("priors<-")
+export()
```

Comment:
Not quite sure where this is coming from but it indicates something isn't correct as none of the exports should be blank

Reply:
>is this relavent with the warning of :
```r
Warning messages:
1: [hazard_link.R:98] Block must have a @name
i Either document an existing object or manually specify with @name 
```

Reply:
O yer that would be the one, `@name` should always have an argument. To be honest I don't think it is needed I think you can just delete it.